src/ui/layout/mct-tree.vue

```diff
@@ -12,6 +12,7 @@
         class="c-tree-and-search__search"
     >
         <search
+            v-show="isSelectorTree"
```

Comment:
we only want to show search in the tree if the user is using a form dialog (e.g., they clicked the "Create" button and want to filter the tree)